src/blocks/table-of-contents/view.js

```diff
@@ -107,6 +108,7 @@ function onScroll() {
  */
 function initializeProgressIndicator() {
 	window.addEventListener( 'scroll', onScroll );
+	window.addEventListener( 'hashchange', updateJumplistActiveItem );
```

Comment:
I think this is going to conflict with the hash fragments I'm using for slides in #50, where the hash fragment will not necessarily correspond to an TOC section on the page.

Reply:
@goldenapples I don't think it will, because this simply uses the hash change as a trigger to check what's visible on-page. There is no interaction with the hash fragment itself.